tests/integ/modin/index/test_index_methods.py

```diff
@@ -18,63 +18,71 @@
 ]
 
 
-@sql_count_checker(query_count=0)
-@pytest.mark.parametrize("native_index", NATIVE_INDEX_TEST_DATA)
-def test_index_astype(native_index):
```

Comment:
did you get rid of this test on purpose?

Reply:
I added it back and modified it a bit